jeni.py

```diff
@@ -399,6 +398,12 @@ class Injector(BaseInjector):
 
             with Injector() as injector:
                 injector.get('injector')
+
+        Local values can be provided when constructing the injector::
+
+            with RequestInjector(request=request) as injector:
+                injector.get('form:username')
```

Comment:
This doc example doesn't make sense without more context. A more trivial illustration would be clearer:

```python
with Injector(foo='bar') as injector:
    injector.get('foo') # 'bar'
```